Tests/Tests/STPShippingAddressViewControllerTest.m

```diff
@@ -0,0 +1,73 @@
+//
+//  STPShippingAddressViewControllerTest.m
+//  StripeiOS Tests
+//
+//  Created by Cameron Sabol on 8/7/18.
+//  Copyright © 2018 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <Stripe/Stripe.h>
+#import "NSLocale+STPSwizzling.h"
+#import "STPFixtures.h"
+
+@interface STPShippingAddressViewControllerTest : XCTestCase
+
+@end
+
+@implementation STPShippingAddressViewControllerTest
+
+- (void)testPrefilledBillingAddress_removeAddress {
+    STPPaymentConfiguration *config = [STPFixtures paymentConfiguration];
+    config.requiredShippingAddressFields = [NSSet setWithObject:STPContactFieldPostalAddress];
+
+    STPAddress *address = [STPAddress new];
+    address.name = @"John Smith Doe";
+    address.phone = @"8885551212";
+    address.email = @"foo@example.com";
+    address.line1 = @"55 John St";
+    address.city = @"Harare";
+    address.postalCode = @"10002";
+    address.country = @"ZW";
+
+    STPShippingAddressViewController *sut = [[STPShippingAddressViewController alloc] initWithConfiguration:config
+                                                                                                      theme:[STPTheme defaultTheme]
+                                                                                                   currency:nil
+                                                                                            shippingAddress:address
+                                                                                     selectedShippingMethod:nil
+                                                                                       prefilledInformation:nil];
+
+    [sut loadView];
+    [sut viewDidLoad];
+}
+
+- (void)testPrefilledBillingAddress_addAddress {
+    [NSLocale stp_setCurrentLocale:[NSLocale localeWithLocaleIdentifier:@"en_ZW"]];
```

Comment:
WDYT about adding assertions (or a separate test?) that `postalCodeIsRequiredForCountryCode:` returns the expected values?

If ZW starts requiring a postal code, or we stop requiring it for US, we start testing a different code path, and the code could break silently.

It also helps document what's being tested, and why those countries were chosen.